### PR TITLE
Added read / write bytes methods

### PR DESCRIPTION
**Pull request by etorreborre**

### CODE REVIEW

File: src/main/scala/com/ambiata/poacher/hdfs/Hdfs.scala [hunk 1]
```diff
@@ -170,6 +170,9 @@ object Hdfs {
     })
   } yield a
 
+  def readBytes(p: Path, chunksize: Int = Streams.DefaultChunkSize): Hdfs[Array[Byte]] =
```
Comment (markhibberd) [inline src/main/scala/com/ambiata/poacher/hdfs/Hdfs.scala]: Unused default arg. Reinforces why we shouldn't do this :)  Given the changes coming down (i.e. https://github.com/ambiata/poacher/compare/topic/path#diff-b04ff8df1fc8bb69319615cccb969153R120) I would just remove the argument and accept a fixed chunk size for now (this default arg is being removed from mundane on same branch).

**Comment by markhibberd** [comment]
:+1:

File: src/main/scala/com/ambiata/poacher/hdfs/Hdfs.scala (hunk 1 above)
Comment (etorreborre) [inline src/main/scala/com/ambiata/poacher/hdfs/Hdfs.scala]: Right, I'm happy with that.

**Comment by charleso** [comment]
:+1:

**Comment by charleso** [comment]
Wondering at some point whether we want versions for `Buffer` as well, but I guess we add them if/when we ever need them.

**Comment by etorreborre** [comment]
@charleso I'm not sure that we'll ever want to have `readBuffer`. `readBytes` is useful in tests to read small files whereas `Buffer` is generally used when streaming data.

**Pull request merged**